OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/dialog/EditSubkeyDialogFragment.java

```diff
@@ -75,7 +75,22 @@ public void onClick(DialogInterface dialog, int which) {
                         sendMessageToHandler(MESSAGE_REVOKE, null);
                         break;
                     case 2:
```

Comment:
While you're touching this code, can you introduce constants for this switch statement? i.e. `LIST_ITEM_CHANGE_EXPIRY`